Unknown error (please report): <type 'exceptions.AttributeError'>
Its value is: AttributeError("type object 'plm.universe.Direction' has no attribute 'west'",)
  File "<script>", line 5, in <module>
